vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -99,6 +99,13 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
         exhibition.setStartSpace(startSpace);
         exhibition.setTitle(title);
         exhibition.setMode(exhibitMode);
+        
+        if(languages.isEmpty()) {
```

Comment:
I believe you can simply make the RequestParam `exhibitLanguage` required instead. Have you tried that?

Reply:
yes, I checked that - the RequestParam exhibitLanguage is required by default and it was giving the server error. 

Can I make it required = false and add a null check here to give out an error message in the ui or remove this if block and keep it as before?

Reply:
Given that there are quite a few request parameter by now, I would create a class the paramters and validate that. Similar to how it's done here:
https://www.baeldung.com/spring-mvc-form-tutorial